### ISSUE TITLE
Uncovering Missing Parameters for SVT-AV1

### ISSUE THREAD

**Issue by RaijinXBlade** [staxrip/staxrip#1150]
Hello there. I have been dabbling around with StaxRip lately, and doing some tests with AV1 and so on. It's simply great to have such a nice GUI for this type of thing.
Anyway, from what I have seen, for the SVT encoder, within the GUI settings, it seems there are a number of missing parameters still. I referenced this https://gitlab.com/AOMediaCodec/SVT-AV1/-/blob/master/Docs/Parameters.md
Unless I am missing something, there doesn't seem to be a way to use certain ones.

So in short, if it's no trouble, it would be nice to have access to all of these remaining parameters, whenever possible. Thanks, and sorry if I have missing something somewhere.

**Comment by Dendraspis** [comment]
From time to time parameters are added and modified, sadly it's hard to keep up. Because `Svt-Av1` is not that that often used within StaxRip I might have skipped some special parameters when re-implementing it.
If there are some params that are missing and you use often, post them with their belonging link from the docs and there is a high chance I will add them.
In case you just wanna play with some params, please use the `Custom` section in the encoder options to add whatever you want.

**Comment by Dendraspis** [comment]
`--enable-qm` is already solved in #1122 and will be available with the next release.

If you have more specific issues, please reopen this issue or open a new one, thanks.
